packages/camera/camera_web/pubspec.yaml

```
@@ -2,7 +2,7 @@ name: camera_web
 description: A Flutter plugin for getting information about and controlling the camera on Web.
 repository: https://github.com/flutter/packages/tree/main/packages/camera/camera_web
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.3.1+4
+version: 0.4.0
```

```suggestion
version: 0.3.2
```

I think the change in behavior is subtle, and hope it's not a breaking change?

(PS @maciejbrzezinski, what would you think if you got this change automatically on your web app, would it break?)

---

I agree, that the change is subtle and in my case it wouldn't break the app, I do not need the audio permission at all